src/modelize/modelize_property.nit

```diff
@@ -880,6 +966,14 @@ redef class AMethPropdef
 		var mysignature = self.mpropdef.msignature
 		if mysignature == null then return # Error thus skiped
 
+		# Check
+		if nsig != null then
+			if not nsig.check_signature(modelbuilder, mclassdef) then
+				self.mpropdef.msignature = null # invalidate
```

Comment:
If the idea is that the `null` will get caught later and will generate an error, the comment should state that.